app/javascript/multi-stamp/components/MultiStamp.vue

```diff
@@ -257,7 +264,7 @@ export default {
       return [
         checkSize(12, 8),
         checkDuplicates(currPlates),
-        // checkExcess(this.excessTransfers)
```

Comment:
Not sure what the story is here...

`checkExcess` seems to be defined in `plateScanValidators` and not used - so could probably be deleted along with its test?

It's a bit of a rabbit hole though as I wonder whether `this.excessTransfers` and related code should also be removed.

Reply:
Reading the comment above the validator code, this check is intended to flag up when you are combining plates and there are more samples in the sources than will fit in the destination. e.g. plate 1 has 50, plate 2 has 50, it flags up that 4 in plate 2 will not fit in the destination (assuming 96).
Instead it now seems to just be comparing `validTransfers` to the number of destination plate wells to do that check in `this.excessTransfers`.
So I think the validator can be removed but not the excessTransfers.